website/addons/wiki/static/pagedown-ace/Markdown.Editor.js

```diff
@@ -111,6 +112,10 @@ var Range = ace.require('ace/range').Range;
             win: 'Ctrl-Y|Ctrl-Shift-Z',
             mac: 'Command-Y|Command-Shift-Z',
         },
+        help: {

```

Comment:
This actually was necessary, I'm getting `TypeError: keyStrokes[identifier] is undefined` in the console without the `help` key in here. Something like:
```
help: {
    win: '',
    mac: '',
},
```